omniNotes/src/androidTest/java/it/feio/android/omninotes/screens/Home.java

```diff
@@ -48,4 +57,71 @@ public static void clickTextNoteButton() {
     public static boolean isReminderSetDisplayed() {
         return HelperMethods.isObjectDisplayed(withId(R.id.card_layout));
     }
+
+    public static void deletePayTheBillsNote() {
+        HelperMethods.deleteElement(withId(R.id.root));
+    }
+
+    public static boolean isArchiveButtonDisplayed() {
+        return HelperMethods.isObjectDisplayed(allOf(withId(R.id.title), withText(Constants.ARCHIVE)));
+    }
+
+    public static boolean isArchiveButtonSiblingWithNotesButton() {
+        Matcher<View> archiveButton = allOf(withChild(withChild(allOf(withId(R.id.title), withText(Constants.ARCHIVE)))), hasSibling(withChild(withChild(allOf(withId(R.id.title), withText(Constants.NOTES))))));
```

Comment:
You don't need "archiveButton" , you can directly return your value.

Reply:
I know, but i thought that would be easier to read it like that.